### ISSUE TITLE
Missing focus indicator for Tabs component

### ISSUE THREAD

**Issue by darekkay** [reach/reach-ui#134]
The [Tabs component](https://ui.reach.tech/tabs) is missing a focus indicator. Either the tabs navigation bar or the currently selected tab itself has to be highlighted when it receives the keyboard focus ([WCAG 2.4.7](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-focus-visible).

Currently, only the *state* of the Tab component is highlighted, i.e. the *selected* tab (whether or not it is actually focused):

![image](https://user-images.githubusercontent.com/3101914/55324998-02bf0480-5484-11e9-92a4-f4bd9a21c925.png)

**Comment by mrm007** [comment]
To add to this, according to [WCAG §2.4.3] and [WAI-ARIA §2.3.1] focus **must** be managed on the tablist role (via `tabindex` or other means). Currently you can use the `Tab` key to get the focus on the container _**in Chrome**_, but not in Firefox.

[WCAG §2.4.3]: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=243#focus-order
[WAI-ARIA §2.3.1]: https://w3c.github.io/aria/#managingfocus

**Comment by chancestrickland** [comment]
@darekkay Can you please expand on this? The user-agent stylesheet applies focus styles by default, and we do not remove them in our default styles. We also don't add anything in an effort to keep the lib as low-level as possible since users will presumably want to style our components differently. But I'm seeing standard UA focus indicators across browsers.

@mrm007 I also don't have any issues tabbing to the container in FF. Can you confirm this is still an issue and, if so, tell me what env you're on so I can try to replicate it?

**Comment by darekkay** [comment]
@chancestrickland I am using the latest Firefox versions (70.0.1 and 71.0b5) and I just can't get the focus to show, either by clicking, tabbing or forcing the "focus" state in DevTools. This might be a Windows/Firefox issue, as you do not alter the `outline` property 🤷‍♀
